src/clj/game/cards/operations.clj

```diff
@@ -615,7 +615,18 @@
                               :trace {:base 2 :msg "give the Runner 1 tag"
                                       :delayed-completion true
                                       :effect (effect (tag-runner :runner eid 1))}}}}
-
+   
+   "MCA Informant"
+   {:req (req (some #(has-subtype? % "Connection") (all-installed state %))) 
+    :prompt "Choose a connection to host MCA Informant" 
+    :choices {:req #(and (has-subtype? % "Connection") (installed? %))} 
+    :effect (effect (host target (assoc card :zone [:discard]))
+            (system-msg (str "hosts MCA Informant on " (:title target))) 
+            (swap! state update-in [:runner :tag] inc)) 
+    :abilities [{:label "[Click], 2 [Credits]: Trash this card."
+                 :effect (effect (trash card {:cause :ability cost}))}] 
```

Comment:
This ability is not actually costing anything. Needs a costs vector like `:cost [:click 1 :credit 2]`. Also, this would add the ability to MCA Informant itself. The ability needs to go on the host. Wetwork Refit is a template here (and is the card I started from in my failing-so-far efforts to get this card working).